ansible/roles/nginx_vhost/tasks/main.yml

```diff
@@ -62,6 +62,20 @@
   tags:
     - nginx_vhost
 
+- name: set vfragments_suffix when vhost_with_appname_conf is true
```

Comment:
Just for avoid this fact leaking out of this role, can you also `unset` this variable at the end of this file, similar to: ```
- name: unset nginx_paths
  set_fact:
    nginx_paths: ''
  tags:
    - nginx_vhost
```

Reply:
Added and tested, thanks for the feedback